src/main/java/com/mael/payroll/PayrollDBFacade.java

```diff
@@ -30,6 +32,43 @@ public void deleteEmployee(int employeeId) {
         return listOfEmployees;
     }
 
+    public void addUnionMember(int memberId, int employeeId) {
+        this.unionMembers.put(memberId, employeeId);
+    }
+
+    public Employee getUnionMember(int memberId) {
+        if (employeeExistsAndIsMember(memberId)) {
+            return getEmployee(unionMembers.get(memberId));
+        }
+        throw new UnionMemberNotInDBException();
```

Comment:
Is this really true? Does this exception mean there's no member or does it mean an existing member is not part of a union?